Gemfile

```diff
@@ -39,6 +39,7 @@ deps_shared_with_common = %w(
   rubocop-performance
   rubocop-rspec
   rubocop-sorbet
+  sem_version
```

Comment:
Is this a library we want to depend on? It seems there's an issue from 2019 which has never been acknowledged: https://github.com/canton7/sem_version/issues/2

Reply:
> Is this a library we want to depend on? It seems there's an issue from 2019 which has never been acknowledged: [canton7/sem_version#2](https://github.com/canton7/sem_version/issues/2)

That definitely is worrying. I'm happy to remove this dependency and implement semver 2 from scratch as it is quite small.